stable/rabbitmq/templates/NOTES.txt

```diff
@@ -1,54 +1,61 @@
 
 ** Please be patient while the chart is being deployed **
 
-  Credentials:
+Credentials:
 
-    echo Username      : {{ .Values.rabbitmq.username }}
-    echo Password      : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "rabbitmq.fullname" . }} -o jsonpath="{.data.rabbitmq-password}" | base64 --decode)
-    echo ErLang Cookie : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "rabbitmq.fullname" . }} -o jsonpath="{.data.rabbitmq-erlang-cookie}" | base64 --decode)
```

Comment:
@juan131 the idea behind using these `echo` statements was to allow users to copy+paste the commands to the shell and have the expressions inside `$()` evaluated and ready to use. With this change the user will have to selectively copy and execute the commands which IMO is not an improvement

Reply:
I guess it may have sense for expressions inside `$()` but what sense does it have for examples like the one below?

```
echo Username      : {{ .Values.rabbitmq.username }}
```